doc/Pacemaker_Explained/en-US/Ch-Multi-site-Clusters.txt

```diff
@@ -247,67 +267,19 @@ resources before will be treated according to the `loss-policy` you set
 within the `rsc_ticket` constraint.
 
 Before the booth can manage a certain ticket within the multi-site cluster,
-you initially need to grant it to a site manually via `booth client` command.
-After you have initially granted a ticket to a site, the booth mechanism
+you initially need to grant it to a site manually via the `booth` command-line
+tool. After you have initially granted a ticket to a site, `boothd`
 will take over and manage the ticket automatically.  
 
 [IMPORTANT]
 ====
-The `booth client` command line tool can be used to grant, list, or
-revoke tickets.  The `booth client` commands work on any machine where
-the booth daemon is running. 
-
-If you are managing tickets via `Booth`, only use `booth client` for manual
-intervention instead of `crm_ticket`. That can make sure the same ticket
+The `booth` command-line tool can be used to grant, list, or
+revoke tickets and can be run on any machine where `boothd` is running. 
+If you are managing tickets via Booth, use only `booth` for manual
+intervention, not `crm_ticket`. That ensures the same ticket
 will only be owned by one cluster site at a time.
 ====
 
-Booth includes an implementation of
-http://en.wikipedia.org/wiki/Paxos_algorithm['Paxos'] and 'Paxos Lease'
-algorithm, which guarantees the distributed consensus among different
-cluster sites. 
-
-[NOTE]
-====
-`Arbitrator`
-
-Each site runs one booth instance that is responsible for communicating
-with the other sites. If you have a setup with an even number of sites,
-you need an additional instance to reach consensus about decisions such
-as failover of resources across sites. In this case, add one or more
-arbitrators running at additional sites. Arbitrators are single machines
-that run a booth instance in a special mode. As all booth instances
-communicate with each other, arbitrators help to make more reliable
-decisions about granting or revoking tickets.
-
-An arbitrator is especially important for a two-site scenario: For example,
-if site `A` can no longer communicate with site `B`, there are two possible
-causes for that:
-
-- `A` network failure between `A` and `B`.
-
-- Site `B` is down.
-
-However, if site `C` (the arbitrator) can still communicate with site `B`,
-site `B` must still be up and running. 
-
-====
-
-===== Requirements =====
-
-- All clusters that will be part of the multi-site cluster must be based on Pacemaker.
-
```

Comment:
On 03/02/2015 05:18 PM, Andrew Beekhof wrote:
>> -if site `A` can no longer communicate with site `B`, there are two possible
>> -causes for that:
>> -
>> -- `A` network failure between `A` and `B`.
>> -
>> -- Site `B` is down.
>> -
>> -However, if site `C` (the arbitrator) can still communicate with site `B`,
>> -site `B` must still be up and running. 
>> -
>> -====
>> -
>> -===== Requirements =====
>> -
>> -- All clusters that will be part of the multi-site cluster must be based on Pacemaker.
>> -
> 
> These two requirements seem worth highlighting

Will submit this and the other changes once we hear from Yan.